### ISSUE TITLE
BytesValue in Any rejects unpadded and URL-safe base64

### ISSUE THREAD

**Issue by coryan** [googleapis/google-cloud-rust#6714]
Unconfirmed.  This is the result of a search with Claude, may be a false positive.

In `src/wkt/src/wrappers.rs`, `BytesValue` deserialization from `Any` JSON representation decodes the string using standard base64 engine:

https://github.com/googleapis/google-cloud-rust/blob/f02b42c2ac5720334cd2b5335824bd4166264527/src/wkt/src/wrappers.rs#L340-L349

```rust
STANDARD
    .decode(s)
    .map(BytesValue::from)
    .map_err(crate::AnyError::deser)
```

The Proto3 JSON specification specifies that byte fields must accept either standard with padding or URL-safe with/without padding base64 encoding.

Using `STANDARD` rejects valid Proto3 JSON payloads that contain unpadded base64 or URL-safe base64 characters (`-` and `_`).
